src/app/modules/audience/views/technology/devices-overview/devices-overview.config.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { ReportDataBaseConfig, ReportDataConfig, ReportDataSection } from 'shared/services/storage-data-base.config';
+
+@Injectable()
+export class DevicesOverviewConfig extends ReportDataBaseConfig {
+  constructor(_translate: TranslateService) {
+    super(_translate);
+  }
+  
+  public getConfig(): ReportDataConfig {
+    return {
+      [ReportDataSection.table]: {
+        fields: {
+          'device': {
+            format: value => value,
+          },
+          'count_plays': {
+            format: value => value,
+          },
+          'sum_time_viewed': {
```

Comment:
@diamond-darrell  - We probably don't need sum_time_viewed, count_loads, load_play_ratio and avg_view_drop_off fields.

Reply:
yes, I was going to remove redundant properties once I get to the tables, but I can do it now